tests/common.go

```diff
@@ -285,6 +297,13 @@ func InitInstance() {
 	} else {
 		logrus.Debugf("Not all information to connect to testrail is provided, skipping updates to testrail")
 	}
+
+	if jiraUserName != "" && jiraToken != "" {
+		jirautils.Init(jiraUserName, jiraToken)
+
+	} else {
+		logrus.Debugf("Not all information to connect to JIRA is provided.")
```

Comment:
We should fail here in my opinion instead of proceeding.

Reply:
We do not want to block any execution due to this. it is completely optional for the user if test needs to raise a issue or not